db/migrate/20250127114020_create_rides.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRides < ActiveRecord::Migration[7.2]
+  def change
+    create_table :rides do |t|
+      t.integer :passenger_id
```

Comment:
You can do like this:
`  t.references :passenger, foreign_key: { to_table: :users }`